src/SearchAction.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace strongsquirrel\actions;
+
+use yii\base\Action;
+use yii\base\InvalidConfigException;
+use yii\base\Model;
+
+/**
+ * Class SearchAction
+ *
+ * @package strongsquirrel\actions
+ */
+class SearchAction extends Action
+{
+    const HTTP_METHOD_GET = 'get';
+    const HTTP_METHOD_POST = 'post';
+
+    /**
+     * @var string
+     */
+    public $scenario = Model::SCENARIO_DEFAULT;
+
+    /**
+     * @var string the name of the view action.
+     */
+    public $view = 'search';
+
+    /**
+     * @var string
+     */
+    public $httpMethod = self::HTTP_METHOD_POST;
```

Comment:
I'm sure that $httpMethod should allow only proper verb. For example: I send a GET request but $httpMethod is POST and I think that action should throw exception in such situation.